image/local.go

```diff
@@ -59,11 +59,24 @@ func (f *Factory) NewLocal(repoName string, pull bool) (Image, error) {
 		RepoName:   repoName,
 		Inspect:    inspect,
 		layerPaths: make([]string, len(inspect.RootFS.Layers)),
-		FS:         f.FS,
 		prevOnce:   &sync.Once{},
 	}, nil
 }
 
+func (f *Factory) NewEmptyLocal(repoName string) Image {
+	inspect := dockertypes.ImageInspect{}
+	inspect.Config = &container.Config{
+		Labels: map[string]string{},
+	}
+	return &local{
+		RepoName:   repoName,
+		Docker:     f.Docker,
+		Inspect:    inspect,
+		layerPaths: []string{},
```

Comment:
`nil` slice is more commonly used instead of empty slice (unlike for maps, which must be pre-allocated). This line could be removed.